core/zoe/docs/RightsInZoe.md

```diff
@@ -0,0 +1,38 @@
+## Rights in Zoe and Governing Contracts
+
+In addition to the underlying rights that are escrowed in a contract,
+Zoe introduces two new rights:
+1. An `escrowReceipt` ERTP payment that proves to a governing contract that an offer
+   has successfully escrowed
+2. A `claimPayoff` ERTP payment that can be unwrapped to get a use
+   object that has a `getPayoff` method which returns an array
+   representing the winnings and/or the refund.
+
+
+The quantity of both an `escrowReceipt` and a `claimPayoff` ERTP
+payment has the following format, showing that an offer has been made
+and escrowed:
+
+```js
+{
+  id: {},
+  offerMade: [rule1, rule2, rule3, ...],
+}
+```
+
+Various governing contracts may also want to create further rights,
+such as the right to enter a particular contract instance (an invite).
+
+An invite quantity has the following format:
+
+```js
+{
+  id: {},
+  offerToBeMade: [rule1, rule2, rule3, ...],
```

Comment:
I think it depends on whether the governing contract chooses to enforce it or not. For instance, you could imagine an invite to a swap that does have a specific counter-offer that it will accept and nothing else. Or, it could be an auction that will accept any sort of bid over a minimum. 

Since nothing in this PR or autoswap uses invites, I think I should take this file out for this PR. It will probably change with a PR that does include invites.